src/main/java/uk/ac/diamond/ispyb/api/DataCollectionPlan.java

```diff
@@ -18,6 +18,7 @@
 public class DataCollectionPlan {
     private Long dcPlanId;
     private String name;
+    private Short sampleOrderInPlan;
```

Comment:
This is functionally fine, but is there a reason for using the object form Short (as well as Long and Double) instead of their primitive counterparts (short, long, double)? The latter is much more efficient.